src/configs/sys_config.py

```diff
@@ -158,16 +158,20 @@ def get_digit_five_support(num_users: int, domains: List[str] = DIGIT_FIVE):
 CIAR10_DPATH = "./datasets/imgs/cifar10/"
 
 NUM_COLLABORATORS = 1
-DUMP_DIR = "/mas/camera/Experiments/SONAR/abhi/"
+# DUMP_DIR = "../../../../../../../home/"
+DUMP_DIR = "./"
```

Comment:
Let's make the default dump directory as `/tmp`
The risk with keeping "./" as dump directory is that the `dump_dir` might try to copy itself in `dump_dir` kicking off a recursive cascade